bigframes/dataframe.py

```diff
@@ -610,7 +629,14 @@ def __repr__(self) -> str:
         column_count = len(pandas_df.columns)
 
         with display_options.pandas_repr(opts):
-            repr_string = repr(pandas_df)
+            import pandas.io.formats
+
+            options = (
+                pandas.io.formats.format.get_dataframe_repr_params()  # type: ignore
```

Comment:
We're sure this returns a copy, not the global instance?

Reply:
yes, its a brand new dict, safe to mutate. adding comments to code to communicate this